src/python/ddapp/ikplanner.py

```diff
@@ -219,6 +220,7 @@ def __init__(self, ikServer, robotModel, jointController, handModels):
         self.additionalTimeSamples = 0
         self.useQuasiStaticConstraint = True
         self.pushToMatlab = True
+	self.pushToMatlabIK = True
```

Comment:
The flags intents to separate the solvers for IK pose and IK trajectory, I pushed a new commit with new flag names.